api/SVGForeignObjectElement.json

```diff
@@ -20,7 +20,7 @@
             "version_added": true
           },
           "ie": {
-            "version_added": "9"
+            "version_added": false
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/foreignObject#Browser_compatibility and https://stackoverflow.com/a/19739804 also indicate `<foreignObject>` was not supported in IE.